CourseApp/ConsoleInputOutput.cs

```diff
@@ -0,0 +1,129 @@
+namespace CourseApp
+{
+    using System;
+    using CourseApp.Calculate;
+    using CourseApp.Class;
+
+    public class ConsoleInputOutput
+    {
+        public ConsoleInputOutput()
+        {
+            StartProgram();
```

Comment:
Please do not run methods in construcor, it's not his work.
Constructor should only check input values if they exist and setup fields and properties

Reply:
Fixed